Jenkinsfile

```diff
@@ -327,6 +291,9 @@ pipeline {
         string(name: 'FUNCTIONAL_HARDWARE_MEDIUM_LABEL',
                defaultValue: 'ci_nvme5',
                description: 'Label to use for the Functional Hardware Medium stage')
+        string(name: 'FUNCTIONAL_HARDWARE_MEDIUM_MD_ON_SDD_LABEL',
```

Comment:
SDD -> SSD
```suggestion
        string(name: 'FUNCTIONAL_HARDWARE_MEDIUM_MD_ON_SSD_LABEL',
```

Reply:
Thanks.  It was misspelled in its definition too.